### DISCUSSION TITLE
How to assign different grain boundary mobility to different grains ?

### DISCUSSION THREAD

**Discussion by zengfy-hust** [idaholab/moose#19459] [Q&A Modules: Phase field] [answered]
Hello everyone:
         I would like to make some changes by assign different GB mobility to different grains like https://www.mdpi.com/1996-1944/12/24/4048/htm,which makes the misorientation angle as an  threshold,I looked up in moose repository and found some cases like this ,the GB mobilities .I would like to assign more grains (more than 100),how can I make this ? thank you .any suggesstion would be fine !

**Comment by GiudGiud** [comment]
@laagesen

**Comment by laagesen** [comment, marked as answer]
We don't have anisotropic mobilities exactly like this currently implemented in MOOSE- the most closely related example shows what happens when different grains have different mobiliites:

moose/modules/phase_field/test/tests/GBAnisotropy/test2.i
